firefox-ios/Client/Experiments/Messaging/GleanPlumbMessageManager.swift

```diff
@@ -322,6 +324,30 @@ class GleanPlumbMessageManager: GleanPlumbMessageManagerProtocol {
         )
     }
 
+    private func sanitizeAction(_ unsafeAction: String, table: [String: String]) -> String? {
+        let action = table[unsafeAction] ?? unsafeAction
+        if action.contains("://") {
+            return action
+        } else {
+            return nil
+        }
+    }
+
+    private func sanitizeTriggers(_ unsafeTriggers: [String], table: [String: String]) -> [String]? {
+        var triggers = [String]()
+        for unsafeTrigger in unsafeTriggers {
+            guard let safeTrigger = table[unsafeTrigger] else {
+                return nil
+            }
+            triggers.append(safeTrigger)
+        }
+        return triggers
+    }
+
+    private func sanitizeStyle(_ unsafeStyle: String, table: [String: StyleData]) -> StyleData? {
+        table[unsafeStyle]
```

Comment:
```suggestion
        return table[unsafeStyle]
```
nit: we prefer having return statements in the codebase.